README.md

```diff
@@ -178,3 +178,27 @@ and then refactor your code from `GwtEvent` to `Event`).
    import org.gwtproject.event.legacy.shared.EventHandler;
    import org.gwtproject.event.legacy.shared.GwtEvent;
    ```
+
+## Instructions
+
+To build gwt-event:
+
+* run `mvn clean install`
```

Comment:
we already change that to:
```
To build gwt-core:

* run `mvn clean install`

on the parent directory. This will build the artifact and run tests against the JVM, J2CL, and GWT2.
```
we can replace `install` with `verify`. I think, in case someone will build and deploy it locally he knows how.

Reply:
+1 for verify, however we don't have any failsafe tests in most of these (notable example being typedarrays), the vast majority are surefire.